examples/simple-udp/main.go

```diff
@@ -144,7 +145,17 @@ func readWriteLoop(conn net.PacketConn, stop chan struct{}, s *sdk.SDK) {
 			allocate(s)
 
 		case "RESERVE":
-			reserve(s)
+			if len(parts) == 2 {
```

Comment:
Sorry I didn't catch this last time, but it would be a bit more idiomatic to do:

```
if len(parts) != 2 {
  respond()
  continue
}
if dur, err := ... ; err != nil {
  respond()
  continue
}
reserve(s, dur)
```

so that the non-error flow isn't indented quite so much.

Reply:
Ah that is nicer. I'll make that change.